gradle-plugin/src/main/groovy/org/groovymc/modsdotgroovy/gradle/tasks/GatherQuiltPlatformDetails.groovy

```diff
@@ -0,0 +1,59 @@
+package org.groovymc.modsdotgroovy.gradle.tasks
+
+import groovy.transform.CompileDynamic
+import groovy.transform.CompileStatic
+import org.gradle.api.artifacts.Dependency
+import org.gradle.api.tasks.CacheableTask
+import org.gradle.api.tasks.TaskAction
+import org.jetbrains.annotations.Nullable
+
+@CacheableTask
+@CompileStatic
+abstract class GatherQuiltPlatformDetails extends GatherFabricPlatformDetails {
+
+    GatherQuiltPlatformDetails() {
+        this.configurationName.convention('minecraft')
+
+        // Todo: Use non-internal Loom API when available - https://github.com/FabricMC/fabric-loom/issues/982
```

Comment:
https://github.com/FabricMC/fabric-loom/commit/a6547244e96c1e1cbffa6a47719dc69eae66a576

Reply:
Thanks, we'll switch to it once 1.5 is released. I'll consider implementing a try-catch solution to support both Loom 1.4 and 1.5